packages/ButtonGroup/src/ButtonGroup.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+
+import * as sc from "./ButtonGroup.styles";
+import ButtonItem from "./components/ButtonItem";
+
+const propTypes = {
+  /** The content of each of the toggle buttons in the group. */
+  children: PropTypes.node,
+
+  /** To show or hide the icons used for selected or not selected. */
+  hasIcons: PropTypes.bool,
+
+  /** If the button is disabled. */
+  isDisabled: PropTypes.bool,
+
+  /** If the width of the button should span it's parent container (100%). */
+  isFullWidth: PropTypes.bool,
+
+  /** If it will be rendered as a <button> element. If false, a <span> will be rendered via an accessible <RawButton>. */
+  isSemantic: PropTypes.bool,
+
+  /** Callback to be executed when any button is clicked to select the item or activated by keyboard. Typically required.
+   * it will return a list of selected items */
+  onChange: PropTypes.func,
+
+  /** Size of the button (font size, min-height, padding, etc). */
+  size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+};
+
+const defaultProps = {
+  children: null,
+  hasIcons: false,
+  isDisabled: false,
+  isFullWidth: false,
+  isSemantic: true,
+  onChange: () => {},
+  size: ShirtSizes.MEDIUM,
+};
+
+const ButtonGroup = props => {
```

Comment:
nip: can we use `function ButtonGroup` when declaring const function might be hard for the reader to distinguish if it's the beginning of the component or just a declaration inside of it.

Reply:
Hi @nahumzs, it took a while, but I'm back at addressing some of these suggestions... I wanted to confirm what is the preferred approach here. 
Just got a bit confused since there are already a bunch of components that use `const name` as the approach.

In this line the idea is to declare the beginning of a component so it should be declared using the `function ButtonGroup(props){...`?

Reply:
Yes exactly, a lot of content in the community declare components with `const MyComponent = (props) => {}` and its 100% valid. 

To make a visual improvement in paprika it is better to use the `function MyComponent` declaration whenever you start a component. Its just a visual breaking point to enhance the visual experience that's all.

Reply:
Sounds good, 💯 I will apply.